### ISSUE TITLE
.pug (the new name for jade) extension doesn't supported by VS

### ISSUE THREAD

**Issue by DvirH** [Microsoft/nodejstools#1182]
When I create a new nodeJS project, the default view engine is jade. As you know, Jade changed recently to pug. The problem is pug haven't highlighting mechanism in the text editor, and it makes a lot of errors because of the spaces and tab issues.

It will be updated for the next version?

By the way, Is .styl extension also came with nodeJS for VS? If it is, there is also a problem with it. It doesn't get highlight as well.

Except those, great work. Thanks.

**Comment by mjbvz** [comment]
Thank you for the repot.

The jade -> pug work is being tracked by #895. We need to update our project templates to use pug and also update the file extension to activate our highlighting for `.pug` files. 

I believe we support creating `.styl` files, but not currently add syntax highlighting or any sort of intellisense. Supporting that is out of scope for NTVS at the moment.

Closing as duplicate of #895. Let's see if we can get the basic highlighting unblocked soon, even if we need more time to update the templates.
